Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -425,8 +463,7 @@ extension ActivityStreamPanel: DataObserverDelegate {
     }
 
     func getHighlights() -> Success {
-        let num = Int(Section.highlights.numberOfItemsForRow())
-        return self.profile.recommendations.getHighlights().both(self.profile.recommendations.getRecentBookmarks(num)).bindQueue(.main) { (highlights, bookmarks) in
+        return self.profile.recommendations.getHighlights().both(self.profile.recommendations.getRecentBookmarks(2)).bindQueue(.main) { (highlights, bookmarks) in
```

Comment:
because we no longer show an entire section of bookmarks we can go back to simply showing 1 or 2. 
The reason we needed to show an entire row before was because we didnt want empty spaces in the section.